src/analysis/retail/priest/holy/modules/talents/BottomRow/AnsweredPrayers.tsx

```diff
@@ -0,0 +1,48 @@
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import TALENTS from 'common/TALENTS/priest';
+import SPELLS from 'common/SPELLS';
+import Events, { HealEvent } from 'parser/core/Events';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import { SpellLink } from 'interface';
+import { ANSWERED_PRAYERS_TRIGGER } from 'analysis/retail/priest/holy/constants';
+
+class AnsweredPrayers extends Analyzer {
+  prayerHeals = 0;
+  healsPerApotheosis = 0;
+
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasTalent(TALENTS.ANSWERED_PRAYERS_TALENT);
+    this.healsPerApotheosis =
+      ANSWERED_PRAYERS_TRIGGER[
+        this.selectedCombatant.getTalentRank(TALENTS.ANSWERED_PRAYERS_TALENT)
+      ];
```

Comment:
This is a bit of an odd way to get the constants, I'd prefer the per-talent number just exist in this class. It's usually how it's done in other modules.

Reply:
Done